Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -566,7 +567,7 @@ extension AVFoundationPlayback {
         if isPaused && isDvrAvailable { return true }
         guard let end = dvrWindowEnd, playbackType == .live else { return false }
         guard let currentTime = player?.currentTime().seconds else { return false }
-        return end > currentTime
+        return end - liveHeadTolerance > currentTime
```

Comment:
Hey @nabilsafatli and @dezouzart , I printed the value of `isDvrInUse` when a positionUpdate is received and even subtracting `liveHeadTolerance`, the property reported that a video was in dvr when the scrubber is on the end of the seekbar. This problem happens when `currentTime` is near `end`.

Reply:
As we talked before, this problem is analyzed in another task.